new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -1839,4 +2062,57 @@ private void changeConfigEmptyPass() throws Exception {
             ADMIN_USERNAME_DEFAULT,
             "");
   }
+
+  private boolean changeListenPort(String domainUid, String domainNS, String setListenPortEnabled) throws Exception {
```

Comment:
You can use oracle.weblogic.kubernetes.utils.WLSTUtils.executeWLSTScript( .. ) for this.

Reply:
I used model in image domain with no t3 config, so it requires extra imp also when will add support for OKE it would be no issue to convert considering that oke does not provide public ip